doc/src/selections.rst

```diff
@@ -44,48 +54,55 @@ false and false to true. ``name(#1) == H and not x(#1) < 5.0`` and ``(z(#2) < 45
-created from a combinaison, just ask!
+List of implemented properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Elisions
---------
+Here is the list of currently implemented properties (either string or numeric
+properties). Additional properties ideas are welcome!
 
-This multiple selection language can be a bit verbose for simpler cases, so it
-is sometimes allowed to remove parts of the selection. First, in the ``atoms``
-context, the ``#1`` variable is optional, and ``atoms: name(#1) == H`` is
-equivalent to ``atoms: name == H``.
+String properties
+-----------------
+
+- ``type``: gives the atomic type;
+- ``name``: gives the atomic name. Some formats store both an atomic name (H3)
+  and an atom type (H), this is why you can use two different selectors
+  depending on the actual data;
+- ``resname``: gives the residue name. If an atom is not in a residue, this
+  return the empty string;
+
+Numeric properties
+------------------
 
-Then, if no context is given, the ``atoms`` context is used. This make ``atoms:
-name == H`` equivalent to ``name == H``.
+- ``index``: gives the atomic index in the frame;
+- ``mass``: gives the atomic mass;
+- ``x``, ``y`` and ``z``: gives the atomic position  in cartesian coordinates;
+- ``vx``, ``vy`` and ``vz``: gives the atomic velocity in cartesian coordinates;
+- ``resid``: gives the atomic residue index. If an atom is not in a residue,
+  this return -1;
 
-And finally, the ``==`` comparison operator is the default one if no operator is
-precised. This means that we can write ``name H`` instead of ``name == H``.
+Supported functions in mathematical expressions are: ``sin``, ``cos``, ``tan``
+for the trigonometric functions; ``asin`` and ``acos`` inverse trigonometric
+functions and ``sqrt``. Adding new functions is easy, open an issue about the
+one you need on the chemfiles repository.
+
+Elisions
+^^^^^^^^
+
+This multiple selection language can be a bit verbose for simpler cases, so it
+is sometimes allowed to remove parts of the selection. The following rules allow
+simpler selections:
+
+- First, in the ``atoms`` context, the ``#1`` variable is optional, and ``atoms:
+  name(#1) == H`` is equivalent to ``atoms: name == H``.
+- Then, if no context is given, the ``atoms`` context is used. This make ``atoms:
+  name == H`` equivalent to ``name == H``.
+- Then if no comparison operator is given, ``==`` is used by default. This means
+  that we can write ``name H`` instead of ``name == H``.
+- Then, multiple values are interpreted as multiple choices. A selection like
+  ``name H O C`` is expanded into ``name H or name O or name C``.
 
 At the end, using all these elisions rules, ``atom: name(#1) == H or name(#1) ==
-O`` is equivalent to ``name H or name O``; and ``bond: name(#1) == C and
-mass(#2) == 4.5`` is equivalent to ``bond: name(#1) C and mass(#2) 4.5``.
+O`` is equivalent to ``name H O``. A more complex example can be ``bonds:
+name(#1) O C and index(#2) 23 55 69``, which is equivalent to ``bonds:
```

Comment:
Is there are a property like 'numbonds' for the number of bonds a given atom has?
(Sorry, this is an awkward place for this - just thought of it while reading).

Reply:
No, not yet. But this could be added =) It might be a bit expensive to compute due to the way bonds are stored currently.